apis/users.py

```diff
@@ -393,3 +447,91 @@ def reset_password():
 
     return "Password Changed", 200
 
+
+
+@user_api.delete("/delete_account")
+@check_user
+def delete_account():
+    """
+        Delete the account of the user in session
+    """
+    
+    user_query1 = """
+    delete from historical_ratings where user_id = %(user_id)s
+    """
+    user_query2 = """
+    delete from ratings where user_id = %(user_id)s
+    """
+    user_query3 = """
+    delete from sprint_runs where user_id = %(user_id)s
+    """
+    user_query4 = """
+    delete from marathonruns where user_id = %(user_id)s
+    """
+    user_query5 = """
+    delete from lobby_runs where user_id = %(user_id)s
+    """
+    user_query6 = """
+    delete from user_lobbys where user_id = %(user_id)s
+    """
+    user_query7 = """
+    delete from achievements_progress where user_id = %(user_id)s
+    """
+    user_query8 = """
+    delete from users where user_id = %(user_id)s
+    """
+    
+    get_hosted_lobbies_query = """
+    select lobby_id from user_lobbys 
+    WHERE user_id=%(user_id)s AND owner = 1
+    """
+    
+    id = session["user_id"]
+    args = {"user_id": id}
+
+    db = get_db()
+    with db.cursor(cursor=DictCursor) as cursor:
+        
+        count = cursor.execute(get_hosted_lobbies_query, args)
+        
+        lobbies = cursor.fetchall()
+        
+        if (count > 0):
```

Comment:
Rather than doing `... WHERE user_lobbys.lobby_id IN ...` you should probably use `cursor.executemany`

Reply:
Fixed